File: 6kyi/multiples_of_3_or_5.py
#!/usr/bin/env python3
# https://www.codewars.com/kata/514b92a657cdc65150000006


def solution(number):
    counter = 0
    for item in range(0, number):
        if item % 3 == 0 or item % 5 == 0:
            counter += item
    return counter

File: codewars/6kyi/multiples_of_3_or_5.lua
#!/usr/bin/env lua
-- https://www.codewars.com/kata/514b92a657cdc65150000006

local solution = {}

function solution.solution(value)
  local result = 0
  for c = value - 1, 1, -1 do
    if c % 3 == 0 then
      result = result + c
    elseif c % 5 == 0 then
      result = result + c
    end
  end
  return result
end

return solution